roles/setup-pxe/tasks/dhcp.yml

```diff
@@ -0,0 +1,30 @@
+
+- name: install DHCP service
+  yum:
+    name:
+      - dhcp-server
+    state: present
+  become: yes
+  tags: dhcp
+
+- name: stop DHCP Server
+  service:
+    name: dhcpd
+    state: stopped
+  become: yes
+  tags: dhcp
+
+- name: Create DHCP config
+  template:
+    src: dhcpd.conf.j2
+    dest: /etc/dhcp/dhcpd.conf
+  become: yes
+  tags: dhcp
```

Comment:
Wouldn't it make sense to create a handler that restarts the dhcpd service and notify this handler when the deciding.conf file is changed ?

Reply:
i was thinking on removing the DHCP from the current flow
convert to `dnsmasq` instead of DHCP and TFTP